ui/src/app/pages/demographic-survey.tsx

```diff
@@ -129,10 +132,14 @@ export const DemographicSurvey = (props: WithSpinnerOverlayProps) => {
                     {Object.keys(errors).map((key) => (
                       <li key={errors[key][0]}>{errors[key][0]}</li>
                     ))}
-                    <li>
-                      You may select "Prefer not to answer" for each unfilled
-                      item listed above to continue
-                    </li>
+                    {Object.keys(errors).some((error) =>
+                      possiblePreferNotToAnswerErrors.includes(error)
+                    ) && (
+                      <li>
+                        You may select "Prefer not to answer" for each unfilled
```

Comment:
this will be displayed for a mix of possiblePNTA and not, so "each" won't be strictly true in this case

Reply:
Good call. I will have to think over the wording.